modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectEntryModelIndexerWriterContributor.java

```diff
@@ -63,6 +64,17 @@ public long getCompanyId(ObjectEntry objectEntry) {
 		return objectEntry.getCompanyId();
 	}
 
+	@Override
+	public IndexerWriterMode getIndexerWriterMode(ObjectEntry objectEntry) {
```

Comment:
Carol also implemented this method in ecac6c4 without the if logic, what do you think should be the final code?